[2025-01-06T17:08:52.430+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:08:52.456+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.unpack_kaggle_data manual__2025-01-06T17:08:39.941636+00:00 [queued]>
[2025-01-06T17:08:52.475+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.unpack_kaggle_data manual__2025-01-06T17:08:39.941636+00:00 [queued]>
[2025-01-06T17:08:52.477+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:08:52.500+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): unpack_kaggle_data> on 2025-01-06 17:08:39.941636+00:00
[2025-01-06T17:08:52.508+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=235) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:08:52.510+0000] {standard_task_runner.py:72} INFO - Started process 236 to run task
[2025-01-06T17:08:52.512+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'unpack_kaggle_data', 'manual__2025-01-06T17:08:39.941636+00:00', '--job-id', '820', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpgl5qd4w8']
[2025-01-06T17:08:52.514+0000] {standard_task_runner.py:105} INFO - Job 820: Subtask unpack_kaggle_data
[2025-01-06T17:08:52.588+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.unpack_kaggle_data manual__2025-01-06T17:08:39.941636+00:00 [running]> on host 97bbb29a7501
[2025-01-06T17:08:52.754+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='unpack_kaggle_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:08:39.941636+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:08:39.941636+00:00'
[2025-01-06T17:08:52.756+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:08:52.823+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-06T17:08:52.839+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:08:52.840+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=unpack_kaggle_data, run_id=manual__2025-01-06T17:08:39.941636+00:00, execution_date=20250106T170839, start_date=20250106T170852, end_date=20250106T170852
[2025-01-06T17:08:52.887+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:08:52.934+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T17:08:52.939+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
